src/index.js

```diff
@@ -33,32 +33,34 @@ https://github.com/solid/solid
  * @main solid-client
  */
 
-var rdf = require('./lib/util/rdf-parser')
-var acl = require('solid-permissions')
-var webClient = require('solid-web-client')(rdf)
-var ClientAuthTLS = require('solid-auth-tls')
-var auth = new ClientAuthTLS(webClient)
-var identity = require('./lib/identity')
-var ns = require('solid-namespace')(rdf)
+const rdf = require('./util/rdf-parser')
+const webClient = require('solid-web-client')(rdf)
+const ClientAuthTLS = require('solid-auth-tls')
+const auth = new ClientAuthTLS(webClient)
+const identity = require('./identity')
+const ns = require('solid-namespace')(rdf)
+const acl = require('solid-permissions')
+Object.assign(acl, acl.Authorization.acl)
+acl.ALL_MODES = acl.Authorization.ALL_MODES
```

Comment:
I'm of the opinion that it's not really `solid-client`'s responsibility to re-export the libraries it uses, and _especially_ that it shouldn't mutate the APIs of the libraries which it uses.

So in this example, if `solid-permissions` should export an `ALL_MODES` constant at the top level, then it's the responsibility of `solid-permissions` to do so. Though I think that namespacing within `acl.Authorization` makes sense.